File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L2/08/Level2.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level2 {
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val `in` = Scanner(System.`in`)
        val angka = `in`.nextInt()
        balik(angka)
    }

    //fungsi membalik
    fun balik(angka: Int) {
        var angka = angka
        while (angka != 0) {
            val sisa = angka % 10
            print(sisa)
            angka = angka / 10
        }
        println()
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L2/09/Level2.ts
// IR/*
// * To change this license header, choose License Headers in Project Properties.
// * To change this template file, choose Tools | Templates
// * and open the template in the editor.
// */

/**
 *
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
function main(): void {
    console.log("Miles\t\tKilometers");
    console.log("-------------------------------");

    // ulang sebanyak 10x
    let mil: number = 1;
    while (mil <= 10) {
        console.log(`${mil}\t\t${mil * 1.609}`);
        mil++;
    }
}

main();

